static/js/components/PersonalTab.js

```diff
@@ -30,7 +35,7 @@ class PersonalTab extends React.Component {
       <Grid className="profile-tab-grid">
         <Cell col={1} />
         <Cell col={10}>
-          <PersonalForm {...this.props} />
+          <PersonalForm {...this.props} validator={personalValidation} />
```

Comment:
ok i think i see what's going on. looks like `<EmploymentForm>` (just as an example) gets a different validator depending on the parent component. makes sense to keep this as is